.buildkite/tools/gh-release.sh

```diff
@@ -82,9 +82,10 @@ response=$(
 
 upload_url="$(echo "$response" | jq -r .upload_url | sed -e "s/{?name,label}//")"
 
+content_type="Content-Type:$(file --mime-type -b "${asset}")"
```

Comment:
can we make this into a nicer error?
Like 
https://github.com/sorbet/sorbet/blob/5a9d35b314d4680797f8a7a79d5ec22427240390/tools/toolchain/webasm-linux/emcc.sh#L11